libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/model/DBColumnGroupElement.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2024 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.tools.dbbrowser.model;
+
+import java.util.List;
+
+import com.oceanbase.tools.sqlparser.statement.common.ColumnGroupElement;
+
+import lombok.Data;
+
+/**
+ * @author: liuyizhuo.lyz
+ * @date: 2024/5/10
+ */
+@Data
+public class DBColumnGroupElement {
```

Comment:
Is it necessary to add `@EqualsAndHashCode` annotation here? I see you used the comparison of this object in `DBTableEditor`

Reply:
The @Data annotation contains @EqualsAndHashcode already~